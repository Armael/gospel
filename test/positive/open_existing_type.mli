type 'a t = 'a list

(*@ open Set *)

(* EXPECTED
   [0] OK
*)
