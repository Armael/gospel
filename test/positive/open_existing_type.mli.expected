
*******************************
********** Parsed file ********
*******************************
type 'a t = 'a list
[@@@gospel {| open Set |}]

*******************************
****** GOSPEL translation *****
*******************************
(*@ open Gospelstdlib *)

type 'a t = 'a list
  

(*@ open Set *)

*******************************
********* Typed GOSPEL ********
*******************************
module open_existing_type.mli

  Namespace: open_existing_type.mli
    Type symbols
      ('a) t [='a list]
      
    Logic Symbols
      
    Field Symbols
      
    Exception Symbols
      
    Namespaces
      
    Type Namespaces
      
  Signatures
    (*@ open Gospelstdlib *)
    
    type 'a t = 'a list
         
    
    (*@ open Set *)

OK
